spec/cypress/integration/accessibility.js

```diff
@@ -27,5 +27,16 @@ describe("Accessibility", () => {
         });
       });
     });
+
+    it("Visit all mentor materials to check for accessiblity", () => {
+      cy.app("load_seed");
+      cy.login("mentor");
```

Comment:
We should probably log in as admin, mentors won't (in principle) see all of them.

That said, do we also need to login as admin in the lesson part test, now that we correctly restrict access to them?